pkgs/desktops/gnome-3/default.nix

```diff
@@ -186,6 +186,8 @@ lib.makeScope pkgs.newScope (self: with self; {
 
   gnome-nettool = callPackage ./apps/gnome-nettool { };
 
+  gnome-network-displays = callPackage ./apps/gnome-network-displays { };
```

Comment:
This should be removed. And the commit message changed.

Reply:
:+1:.